### PR TITLE
Update E2E suite instructions

### PR DESCRIPTION
**Pull request by pivotaljohn**

### CODE REVIEW

File: test/e2e/README.md [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
+$ export IMGPKG_E2E_RELOCATION_REPO=index.docker.io/k8slt/imgpkg-test-relocation
+```
+
+#### Execute Specific Test(s)
+
+Test scripts pass down arguments ultimate to the `go test` invocation.
+Specify which tests to run using the `-run` `go test` argument:
+
+```bash
+$ ./hack/test-e2e.sh -run *Copy*FromATar*
```
Comment (DennisDenuto) [inline test/e2e/README.md]: hmmm I tried this (using zsh) and it failed with:
```
zsh: no matches found: *Copy*FromATar*
```

`./hack/test-e2e.sh -run "*Copy*FromATar*"` failed with:
```
testing: invalid regexp for element 0 of -test.run ("*Copy*FromATar*"): error parsing regexp: missing argument to repetition operator: `*`
```

This worked though!:
```
./hack/test-e2e.sh -run ".*Copy.*FromATar.*"
```

File: test/e2e/README.md [hunk 2]
```diff
@@ -1,10 +1,47 @@
...
+```
+
+or
+
+```bash
+$ ./hack/test-all-local-registry.sh 5001 -run *Copy*FromATar*
```
Comment (DennisDenuto) [inline test/e2e/README.md]: Similar issue as the above test-e2e.sh

`*Copy*FromATar*` -> `".*Copy.*FromATar.*"`

**Comment by DennisDenuto** [comment]
apart from the 2 comments left, this looks great!! 

Since it is just a README change I don't need extra confidence running the external registry tests

File: test/e2e/README.md (hunk 1 above)
Comment (cppforlife) [reply test/e2e/README.md]: @pivotaljohn wanna update?

Comment (pivotaljohn) [reply test/e2e/README.md]: Yeah... apologies for the delay; dropped off my radar.

Thanks to @joaopapereira for poking me with a stick.

**Comment by pivotaljohn** [comment]
Checks failed. Are unrelated to this PR.

**Comment by joaopapereira** [comment]
yeah we know, we have been having some flaky tests in windows due to the registry that we are using

**Pull request merged**